lib/icinga/dependency-group.cpp

```diff
@@ -1,6 +1,7 @@
 /* Icinga 2 | (c) 2024 Icinga GmbH | GPLv2+ */
 
 #include "icinga/dependency.hpp"
+#include "icinga/service.hpp"
```

Comment:
Is this include a leftover from your previous iterations or it's actually needed for something?

Reply:
No, it's not. Initially, I copy-pasted `IsReachable()` into that file, and that one survived the cleanup. Now it's gone.